manifests/init.pp

```diff
@@ -120,15 +136,41 @@
   Boolean $candlepin_db_ssl = $::katello::params::candlepin_db_ssl,
   Boolean $candlepin_db_ssl_verify = $::katello::params::candlepin_db_ssl_verify,
   Boolean $candlepin_manage_db = $::katello::params::candlepin_manage_db,
+
+  Boolean $manage_candlepin = $::katello::params::manage_candlepin,
+  Boolean $manage_qpid = $::katello::params::manage_qpid,
+  Boolean $manage_pulp = $::katello::params::manage_pulp,
+  Boolean $manage_application = $::katello::params::manage_application,
 ) inherits katello::params {
-  include ::katello::repo
-  include ::katello::candlepin
-  include ::katello::qpid
-  include ::katello::pulp
-  Class['katello::repo'] -> Class['katello::pulp']
-  include ::katello::application
-  Class['katello::repo'] -> Class['katello::application']
-  Class['katello::candlepin'] -> Class['katello::application']
+  $pulp_manage_httpd = ! $manage_application
+  $candlepin_url = "https://${candlepin_hostname}:8443/candlepin"
+  $qpid_url = "amqp:ssl:${qpid_hostname}:5671"
+  $pulp_url = "https://${pulp_hostname}/pulp/api/v2/"
+
+  if $manage_candlepin {
+    include ::katello::candlepin
+  }
+
+  if $manage_qpid {
+    include ::katello::qpid
+  }
+
+  if $manage_pulp {
+    include ::katello::repo
```

Comment:
This can be dropped I think given we are using Pulp's repositories directly now.

Reply:
Should we use pass ```manage_repo``` to the pulp class to get that repository installed?